public/snippet-script.js

```diff
@@ -116,9 +116,10 @@ window.addEventListener('load', () => {
     analytics: {
       baseUrl: 'https://api.staging.empathy.co/statistics/v2'
     },
-    lang,
     instance,
+    lang,
     audience: 'enterprise',
+    playboardUrl: 'https://playboard-platform.staging.empathy.co/empathy',
```

Comment:
Not specifically part of this fix but we also want to include the missing Playboard link in Backroom.